app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -11,9 +12,7 @@ function PersonalKeyStep() {
       <Button isOutline className="margin-right-2 margin-bottom-2 tablet:margin-bottom-0">
         {t('forms.backup_code.download')}
       </Button>
-      <Button isOutline className="margin-right-2 margin-bottom-2 tablet:margin-bottom-0">
-        {t('users.personal_key.print')}
-      </Button>
+      <PrintButton isOutline className="margin-right-2 margin-bottom-2 tablet:margin-bottom-0" />
```

Comment:
Is there a reason that a separate component is preferable to just a `<Button onClick={print}>` with maybe like a shared helper function for `print` that might be easier to mock?

I guess this is easier to match between client/server?

Reply:
> Is there a reason that a separate component is preferable to just a `<Button onClick={print}>` with maybe like a shared helper function for `print` that might be easier to mock?
> 
> I guess this is easier to match between client/server?

Yeah, in my mind it helps provide some extra assurances that the behavior and default label are consistent. The idea of omitting customizable label is intentional to try to enforce some consistency with how these buttons are presented.

Also, components are cheap. I think the bar should be pretty low for when it makes sense to create one.